CallsheetLocalizations/uk.xcloc/Localized Contents/uk.xliff

```diff
@@ -37,1064 +37,1065 @@
-        <source>Choose spoiler-risky things to hide by default.</source>
+        <source>Choose spoiler-risky things to hide by default.</source><target>Оберіть що приховувати за замовчуванням, щоб уникнути спойлерів</target>
         <note>Subtitle for a row in Settings</note>
       </trans-unit>
       <trans-unit id="Clear" xml:space="preserve">
-        <source>Clear</source>
+        <source>Clear</source><target>Очистити</target>
         <note/>
       </trans-unit>
       <trans-unit id="Clear all" xml:space="preserve">
-        <source>Clear all</source>
+        <source>Clear all</source><target>Очистити усе</target>
         <note>Shown when clearing recent searches or search history</note>
       </trans-unit>
       <trans-unit id="Click Me" xml:space="preserve">
-        <source>Click Me</source>
+        <source>Click Me</source><target>Натисни мене</target>
         <note>Do not translate</note>
       </trans-unit>
       <trans-unit id="Close" xml:space="preserve">
-        <source>Close</source>
+        <source>Close</source><target>Закрити</target>
         <note/>
       </trans-unit>
       <trans-unit id="Connect to devices on your local network to show what's playing." xml:space="preserve">
-        <source>Connect to devices on your local network to show what's playing.</source>
+        <source>Connect to devices on your local network to show what's playing.</source><target>Приєднуватися до локальний пристроїв щоб показувати що грає.</target>
         <note>Subtitle in Persnickety Preferences</note>
       </trans-unit>
       <trans-unit id="Copy" xml:space="preserve">
-        <source>Copy</source>
+        <source>Copy</source><target>Копіювати</target>
         <note/>
       </trans-unit>
       <trans-unit id="Copy Error Information" xml:space="preserve">
-        <source>Copy Error Information</source>
+        <source>Copy Error Information</source><target>Копіювати дані помилки</target>
         <note/>
       </trans-unit>
       <trans-unit id="Copy e-mail address" xml:space="preserve">
-        <source>Copy e-mail address</source>
+        <source>Copy e-mail address</source><target>Копіювати електронну адресу</target>
         <note/>
       </trans-unit>
       <trans-unit id="Could not load static JSON" xml:space="preserve">
-        <source>Could not load static JSON</source>
+        <source>Could not load static JSON</source><target>Не можу завантажити статичний JSON</target>
         <note>Do not translate</note>
       </trans-unit>
       <trans-unit id="Could not parse static JSON" xml:space="preserve">
-        <source>Could not parse static JSON</source>
+        <source>Could not parse static JSON</source><target>Не можу прочитати статичний JSON</target>
         <note>Do not translate</note>
       </trans-unit>
       <trans-unit id="Credits" xml:space="preserve">
-        <source>Credits</source>
+        <source>Credits</source><target>Роботи</target>
```

Comment:
@cliss where is this string used? (depending on context, it might be translated as "people who worked on a thing" or as "things that a person worked on")

Reply:
<img width="415" alt="Screenshot 2024-05-13 at 9 48 20 AM" src="https://github.com/cliss/callsheet-localizations/assets/282460/75770a4f-895d-4038-b2c7-3fad16657dac">

Bottom-right; as a header for "does this movie have mid- or post-credits scenes?"

Reply:
Oh, okay